corehq/apps/reminders/templates/reminders/keyword_list.html

```diff
@@ -22,20 +22,58 @@ <h2>{% trans 'Manage Keywords' %}</h2>
       {% trans 'Add Structured Keyword' %}
     </a>
 
+    <p class="pull-right ko-template" id="lock-container" data-bind="if: hasLinkedModels">
+      <!-- ko if: allowEdit -->
+        <span data-bind="ifnot: unlockLinkedData">
+          <button class="btn btn-primary" data-toggle="modal" data-target="#edit-warning-modal">
+            <i class="fa fa-link"></i>
+            {% trans 'Unlock Linked Keywords For Me' %}
+          </button>
+        </span>
+        <span data-bind="if: unlockLinkedData">
+          <button class="btn btn-primary" data-bind="click: toggleLinkedLock">
+            <i class="fa fa-link"></i>
+            {% trans "Lock Linked Keywords For Me" %}
+          </button>
+        </span>
+      <!-- /ko -->
+      <!-- ko ifnot: allowEdit -->
+        <span>
+          <i class="fa fa-link"></i>
+          Contains Linked Roles
+        </span>
+      <!-- /ko -->
+      <span class="hq-help-template"
+          data-placement="left"
+          data-title="{% trans 'Linked Configurations' %}"
+          data-content="
+            {% blocktrans %}
+              Linked configurations are controlled from a separate, linked project space.
+                <a href='https://confluence.dimagi.com/display/commcarepublic/Enterprise+Release+Management' target='_blank'>Learn more</a>.
+            {% endblocktrans %}">
+      </span>
+    </p>
+
   </div>
 {% endblock %}
 
 {% block pagination_templates %}
   <script type="text/html" id="keyword-row-template">
     <td class="col-md-5">
-      <a data-bind="attr: { href: editUrl }, text: keyword"></a>
+      <span data-bind="ifnot: !upstream_id || $root.unlockLinkedData()">
+        <a data-bind="attr: { href: viewUrl }, text: keyword"></a>
+      </span>
+      <span data-bind="if: !upstream_id || $root.unlockLinkedData()">
+        <a data-bind="attr: { href: editUrl }, text: keyword"></a>
```

Comment:
Could these conditionals be added to the `<a>` tag rather than spans?

Reply:
As mentioned above, the conditionals need to be on a parent element. In this case, I don't think I can add it to the containing `<td>` element, because I need both `if` and `ifnot` conditionals, and that would mean multiple `td`s instead of the one we have here.